### PR TITLE
Fix #3981 GFI unavailable when QB window is opened

### LINKED ISSUE #3981: GFI unavailable when the 'Filter Layer' Query Builder window is opened
### Description

In #3785 it was required that:

> Once a valid filter is created the Apply icons is enabled. In order to see it applied on the map I need to click on the Apply Icon in the Toolbar; Once I click on the Apply Icon, the map content should be updated accordingly, no FG is shown. **If I do a GetFeatureInfo, the filter I am creating is applied to the content I get, the QB is not closed so that I can keep working**. Now I can save or discard.

and

> Save enables so I can save my work, **I click on Save Icon in the QB Toolbar. The filter is persisted but the QB is not closed ( I might want to keep playing).**

When applying/saving a filter the GetFeatureInfo is unavailable.

### In case of Bug (otherwise remove this paragraph)

*Browser Affected*
(use this site: https://www.whatsmybrowser.org/ for non expert users)
- [ ] Internet Explorer
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

*Browser Version Affected*

- All

*Steps to reproduce*

- open a map
- load a vector layer
- open the TOC
- select the layer
- open the QueryBuilder by clicking the 'Filter Layer' icon of the TOC toolbar
- set up a filter
- click on 'Apply' in the QB toolbar
- click a point on the map
- verify that GFI works
- click on 'Save' in the QB toolbar
- click a point on the map
- verify that GFI works

*Expected Result*

- GetFeatureInfo window opens

*Current Result*

- Nothing happens (GFI unavailable)

### Other useful information (optional):

More in general, when the QB window is opened the GFI is unavailable (it is the same for the FeatureGrid query builder).

### PR DESCRIPTION
**Pull request by kasongoyo**
## Description
Enable GFI when QB is open

## Issues
 - #3981 
 - ...

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
see #3981 

**What is the new behavior?**
The GFI get opened altogether with QB 

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes, and I documented them in migration notes
 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: web/client/selectors/mapInfo.js [hunk 1]
```diff
@@ -72,13 +70,11 @@ const stopGetFeatureInfoSelector = createSelector(
     measureActiveSelector,
     drawSupportActiveSelector,
     annotationsEditingSelector,
-    queryPanelSelector,
-    (isMapInfoDisabled, isMeasureActive, isDrawSupportActive, isAnnotationsEditing, isQueryPanelActive) =>
+    (isMapInfoDisabled, isMeasureActive, isDrawSupportActive, isAnnotationsEditing) =>
```
Comment (MV88) [inline web/client/selectors/mapInfo.js]: Please check if tests needs to be updated, especially if all cases of stopGetFeatureInfoSelector are covered

Comment (kasongoyo) [reply web/client/selectors/mapInfo.js]: It seems all tests are fine and the change does not negatively affect anything.

Comment (MV88) [reply web/client/selectors/mapInfo.js]: ok

**Comment by tdipisa** [comment]
@kasongoyo please, can you backport this to the stable branch 2019.02.xx?

**Pull request merged**